x/dispute/keeper/keeper.go

```diff
@@ -22,62 +21,45 @@ const (
 
 type (
 	Keeper struct {
-		cdc        codec.BinaryCodec
-		storeKey   storetypes.StoreKey
-		memKey     storetypes.StoreKey
-		paramstore paramtypes.Subspace
+		cdc          codec.BinaryCodec
+		storeService store.KVStoreService
+		Params       collections.Item[types.Params]
 
 		accountKeeper  types.AccountKeeper
 		bankKeeper     types.BankKeeper
 		oracleKeeper   types.OracleKeeper
 		reporterKeeper types.ReporterKeeper
+		Disputes       *collections.IndexedMap[uint64, types.Dispute, DisputesIndex] // dispute id -> dispute
+		OpenDisputes   collections.Item[types.OpenDisputes]
```

Comment:
maybe we can make "OpenDisputes" an index over Disputes?

Reply:
we could but maybe on a different pr